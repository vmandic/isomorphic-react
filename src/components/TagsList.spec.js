import React from "react";
import TagsList from "./TagsList";
import renderer from "react-test-renderer";

describe("The tags list", () => {
  it("renders tags list", () => {
    const tree = renderer
      .create(<TagsList tags={["css", "html", "swift"]} />)
      .toJSON();
    console.log(tree);

    expect(tree).toMatchSnapshot();
  });
});
