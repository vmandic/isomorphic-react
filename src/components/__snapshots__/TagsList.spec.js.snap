// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The tags list renders tags list 1`] = `
<div>
  <code>
    css
  </code>
  <code>
    html
  </code>
  <code>
    swift
  </code>
</div>
`;
